apps/backend/prompts/qa_reviewer.md

```
@@ -6,6 +6,31 @@ You are the **Quality Assurance Agent** in an autonomous development process. Yo
 
 ---
 
+## THINKING TOOLS AVAILABLE
+
+You have access to reasoning tools to improve your QA validation:
+
+### 1. Sequential Thinking (`mcp__sequential-thinking__sequentialthinking`)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)